src/appendices/git/workflows.rst

```diff
@@ -0,0 +1,170 @@
+.. _git-workflows:
+
+.. index::
+   single: Git; workflows
+
+Git Workflows
+=============
+
+This cheatsheet covers basic workflows for working with local and remote Git repositories.
+
+For details on the Git commands mentioned, or on creating and cloning repositories, revisit the chapter on :ref:`git-chapter`.
+
+.. _single-dev-branch:
+
+Single Developer / Single Branch
+--------------------------------
+
+When working on a project by yourself, you will typically follow this workflow:
+
+.. _basic-flow:
+
+#. Make changes to your code
+#. Check status: ``git status``
+#. Stage changes: ``git add .``
+#. Commit changes: ``git commit -m "Added feature X"``
+#. (If working with a remote repository) Push changes:
+   ``git push origin master``
+
+.. _team-single:
+
+Team / Single Branch
+--------------------
+
+The case of a team working on a project with a single branch is *very* uncommon. However, multiple developers working on the same branch happens quite a bit. This workflow, therefore, can be thought of as a sub-workflow of the :ref:`team-feature` workflow.
+
+This workflow is similar to the :ref:`single-dev-branch` workflow, only now you must be mindful to merge in changes made by others.
+
+#. Make changes to your code
+#. Check status: ``git status``
+#. Stage changes: ``git add .``
+#. Commit changes: ``git commit -m "Added feature X"``
+#. Pull changes made by other: ``git pull origin master``
+#. Merge changes as necessary
+#. Push changes: ``git push origin master``
+
+.. _single-feature:
+
+Single Developer / Feature Branches
+-----------------------------------
+
+.. index:: 
+   single: Git; branch
+   single: Git; feature branch
+
+As you become more comfortable with Git and begin to build larger projects on your own, you'll want to use **feature branches**. This is a branch created in order to *isolate* changes related to a *single* feature or bug. Professional developers use them for a few reasons:
+
+- They can work on multiple new features at the same time, while keeping the work separated.
```

Comment:
For screen readers, this needs to be an ol.

Reply:
I think keep this an `ul` is okay. Screen readers can read unordered lists just fine, but semantically, the content should fit the list type. The order of these items isn't important in any way.

Reply:
OK.